.github/actions/macos-setup-env/action.yml

```diff
@@ -15,6 +15,8 @@ runs:
   using: "composite"
   steps:
     - uses: coursier/cache-action@v6
+      with:
+        extra-keys: 
```

Comment:
does this need to be added to the linux case as well? not sure what this line does.